matrix-sdk-android/src/main/java/im/vector/matrix/android/api/session/room/powerlevels/PowerLevelsHelper.kt

```diff
@@ -56,6 +69,26 @@ class PowerLevelsHelper(private val powerLevelsContent: PowerLevelsContent) {
         } else false
     }
 
+    fun canInvite(userId: String): Boolean {
```

Comment:
I do not really know how to improve this, but we may understand that the method return the ability to invite the user passed in parameter, which is wrong. Maybe just add Javadoc to clarify that (applicable to other methods below too)